py4j-java/src/main/java/py4j/CallbackClient.java

```diff
@@ -207,7 +209,7 @@ public CallbackClient(int port, InetAddress address, long minConnectionTime, Tim
 	 */
 	public CallbackClient(int port, InetAddress address, String authToken, long minConnectionTime,
 			TimeUnit minConnectionTimeUnit, SocketFactory socketFactory, boolean enableMemoryManagement,
-			int readTimeout) {
+			int readTimeout, int createSocketTimeout) {
```

Comment:
similar to above, likely want to avoid an API break with another telescoping 🔭 constructor

```
public CallbackClient(int port, InetAddress address, String authToken, long minConnectionTime,
			TimeUnit minConnectionTimeUnit, SocketFactory socketFactory, boolean enableMemoryManagement,
			int readTimeout) {
    this(port, address, authToken, minConnectionTime, minConnectionTimeUnit, socketFactory,
        enableMemoryManagement, readTimeout, GatewayServer.DEFAULT_CONNECT_TIMEOUT)
}
```